src/content/docs/ai-monitoring/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: 'Remove sensitive data with drop filters'
+metaDescription: 'Drop filters prompts AI monitoring to drop attributes containing sensitive data.'
+freshnessValidatedDate: never
+---
+
+import aiDropFilterModal from 'images/ai_screenshot-crop_drop-filter-modal.webp'
+
+import aiDropFilterModalCrop from 'images/ai_screenshot-crop_drop-filter-modal.webp'
+
+Our agents collect and store your data in the New Relic database (NRDB). When an agent captures an interaction with your AI assistant, it stores the contents of those prompts and responses as text string attributes within data events. This gives you greater insight into the cost, accuracy, and performance of your AI-powered app, but you may not want to store interactions that contain sensitive data. We recommend creating drop filters. 
```

Comment:
is there a reason we don't have `LlmVectorSearch` and `LlmVectorSearchResult`? those are the events that are produced in langchain when using vector stores. 

Also missing `LlmChatCompletionSummary`, maybe this one is omitted because it doesn't have any pontential sensitive data.

Reply:
Hm I think when last we were chatting on the drop filters these weren't in the spec yet. I would agree that anything mentioned in [this section](https://source.datanerd.us/agents/agent-specs/blob/f24d0407fb528efd7490f9f5e771e69b63992340/LLMs.md#attributes-to-be-excluded-when-ai_monitoringrecord_contentenabled-is-disabled) of the spec as being ignored when `ai_monitoring.record_content.enabled` is disabled should be called out as something to potentially drop. We'll want to add this to the table in our drop filter UI as well, plus feedback.

You are correct that `LlmChatCompletionSummary` is not included here because it doesn't have any customer inputs/outputs.